reporter/src/funTest/kotlin/reporters/opossum/OpossumReporterFunTest.kt

```diff
@@ -67,10 +67,14 @@ private fun TestConfiguration.generateReport(ortResult: OrtResult): String {
     )
 
     val outputDir = createTestTempDir()
-
     val outputFile = OpossumReporter().generateReport(input, outputDir, emptyMap()).single().inputStream()
 
-    return GzipCompressorInputStream(outputFile)
-        .bufferedReader()
-        .use { it.readText() }
+    var reportAsString = ""
+    ZipInputStream(outputFile).use { zis ->
+        val entry = zis.nextEntry
+        if (entry?.name == "input.json") {
+            reportAsString = zis.readAllBytes().toString(Charsets.UTF_8)
+        }
+    }
```

Comment:
Consider using a similar approach as in https://github.com/oss-review-toolkit/ort/pull/6628/files#diff-1c9eb4cfac3e546739eba0f24ba356c27589747bdfead7fbf65c2b448d25bb5bR429-R430 instead.